### PR TITLE
Test lowest dependencies

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
did you abort the build?

**Comment by lsmith77** [comment]
needs a rebase

File: Tests/WebTest/Admin/Extension/MenuNodeReferrersExtensionTest.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function testEdit()
     {
         $crawler = $this->client->request('GET', '/admin/cmf/menu-test/content/test/content-1/edit');
         $res = $this->client->getResponse();
-        $this->assertEquals(200, $res->getStatusCode());
+        $this->assertResponseSuccess($res);
```
Comment (lsmith77) [inline Tests/WebTest/Admin/Extension/MenuNodeReferrersExtensionTest.php]: this method doesn't seem to be available yet when testing lower

Comment (WouterJ) [inline Tests/WebTest/Admin/Extension/MenuNodeReferrersExtensionTest.php]: Reverted

**Comment by lsmith77** [comment]
hurray!

**Pull request merged**